app/script/notification/NotificationRepository.js

```diff
@@ -832,7 +851,7 @@ z.notification.NotificationRepository = class NotificationRepository {
       return isEventToNotify;
     }
 
-    const isSelfMentioned = messageEntity.is_content() && messageEntity.isUserMentioned(userId);
-    return isEventToNotify && isSelfMentioned;
+    const isSelfMentionOrReply = messageEntity.is_content() && messageEntity.isResponseToUser(userId);
```

Comment:
I guess the condition is a bit off here. Should probably be

```suggestion
    const isSelfMentionOrReply = messageEntity.is_content() && ( messageEntity.isUserMentioned(userId) || messageEntity.isResponseToUser(userId));
```

Reply:
Ok, ignore this comment. It is included in the `isResponseToUser` actually